cmd/podman/build.go

```diff
@@ -52,12 +52,22 @@ func init() {
 	flags := buildCommand.Flags()
 	flags.SetInterspersed(false)
 
-	flags.BoolVar(&layerValues.ForceRm, "force-rm", true, "Always remove intermediate containers after a build, even if the build is unsuccessful. (default true)")
-	flags.BoolVar(&layerValues.Layers, "layers", true, "Cache intermediate layers during build. Use BUILDAH_LAYERS environment variable to override")
 	budFlags := buildahcli.GetBudFlags(&budFlagsValues)
+	flag := budFlags.Lookup("pull-always")
```

Comment:
Please add a comment on why that's necessary.

Reply:
In Podman we are trying to match Docker defaults, based on developers running the tools.  

In Buildah we were trying to optimize for CI/CD Systems.

The force-rm, is actually because podman does not really understand buildah containers.
pull-always might be debatable what the correct default is.
In a CI/CD system you shouldn't need to be creating Layers, since each time the build happens, it happens from scratch.  Whereas when a developer is building a Dockerfile, --layers is critical to make their life easier.